### ISSUE TITLE
Scanned Map batch upload

### ISSUE THREAD

**Issue by jpstroop** [pulibrary/plum#1034]
This came up in our (@eliotjordan, @tampakis, and me)  meeting with Wangyal on Friday and I don't think there is a ticket. I had a few thoughts over the weekend. 

The initial thought was that we should be able to ingest TIFFs in batches, one TIFF per map, with preset defaults for rights, access, location, etc., and that the convention would be `BIBID.tif`. I still agree with that, but it suspect, even though he says it's not the case, that it's only a matter of time before we need more than one image per object. With a little more work I also think this could be very useful for projects that involve outsourcing or other bulk digitization projects (these already happen with the archives and microfilm digitization with some regularity).

Proposal: What if we support two conventions? The simple case (`BIBID.tif`) but also:

```
BIBID1/
  0001.tif
  0002.tif
  ...
BIBID2/
  0001.tif
  0002.tif
  0003.tif
```

How complicated would this get? If we wanted to support doing this for maps and books you'd need to be able to choose the model as well. I think the only requirement the file names is that they sort appropriately. For books, what about MVWs? I'm sure it would come up. Is that:

```
BIBID3/
  v01/
    0001.tif
    0002.tif
  v02/
    0001.tif
    0002.tif
    0003.tif
  ...
```
or is it going to get too complex?

**Comment by escowles** [comment]
This process is for batch ingest of content that doesn't have METS or some other metadata we can extract the structure from, right?  Are we going to have a lot of MVWs that fall into this category?

It doesn't seem to hard to write a rake task that handles the first two scenarios (each file creates a ScannedResource, each directory creates a ScannedResource), and adapt as needed later if we encounter MVW scenarios.

**Comment by jpstroop** [comment]
No METS, correct. I think we already do get MVWs from vendors working on microfilm and fiche digitization. A rake task would be good, but for the initial (`BIBID.tif`) case, the thought was that this  feature would be available via the web UI (right, @eliotjordan ?)

**Comment by jpstroop** [comment]
I worry about validation...conventions are nice, but if someone were to get it wrong we could wind up with a mess.

**Comment by eliotjordan** [comment]
@jpstroop Right, the idea was to enable this via the web UI.

**Comment by escowles** [comment]
So the user would navigate to the directory in BrowseEverything, and then the BrowseEverythingIngester would need to look at the folder structure to decide how to handle the files and what IDs/structure it can pull out of the directory names?  That seems possible, but makes me even more in favor of only building in the complexity we need now.

**Comment by jpstroop** [comment]
Makes sense. I think it's a separate screen from the file manager. Turns out Hyrax has a batch uploader (#1042) so maybe we use that as the point of departure, and start with the easy case, as you suggest?

**Comment by eliotjordan** [comment]
Closed by #1112
